index.js

```diff
@@ -216,6 +216,11 @@ function trimKey(key) {
   return key.replace(/(^\/)|(\/$)/g, '');
 }
 
+var writeOpList = ['POST', 'PUT', 'DELETE'];
+function isWriteOp(method) {
+  return writeOpList.indexOf(method) !== -1;
```

Comment:
Why saving the writeOps as a separate variable with the danger of it getting modified etc? Why not just create it and check it in a single line?